tox.ini

```diff
@@ -114,10 +114,10 @@ commands =
     mpldev: pip install -U --pre -i https://pypi.anaconda.org/scientific-python-nightly-wheels/simple matplotlib
 
     pip freeze
-    !cov-!double: pytest --pyargs astropy {toxinidir}/docs {env:MPLFLAGS} {posargs}
-    cov-!double: pytest --pyargs astropy {toxinidir}/docs {env:MPLFLAGS} --cov astropy --cov-config={toxinidir}/pyproject.toml --cov-report xml:{toxinidir}/coverage.xml {posargs}
+    !cov: pytest --pyargs astropy {toxinidir}/docs {env:MPLFLAGS} {posargs}
+    cov: pytest --pyargs astropy {toxinidir}/docs {env:MPLFLAGS} --cov astropy --cov-config={toxinidir}/pyproject.toml --cov-report xml:{toxinidir}/coverage.xml {posargs}
 
-    double: python -c 'import sys; from astropy import test; test(); sys.exit(test())'
+    double: pytest --pyargs astropy {toxinidir}/docs {env:MPLFLAGS} {posargs}
```

Comment:
@saimn and @astrofrog will this work for the double tests. It just runs all of the tests twice in the same tox environment.